src/pages/aens/NameDetails.vue

```diff
@@ -74,6 +81,7 @@ export default {
     address() {
       return getAddressByNameEntry(this.details);
     },
+    ...mapGetters({ currentNetwork: 'currentNetwork' }),
```

Comment:
```suggestion
    ...mapGetters(['currentNetwork']),
```
https://vuex.vuejs.org/guide/getters.html#the-mapgetters-helper